### DISCUSSION TITLE
Suggestions from user February 2022

### DISCUSSION THREAD

**Discussion by vshcherb** [osmandapp/OsmAnd#13709] [Ideas]
1. Suggestion: easier way to import audio/video notes for offline use
As an example, importing pictures from the web into osmand's audio/video notes plugin is possible but complicated:
1. Download picture
2. Enter geolocation into [osm.org](http://osm.org/) to obtain shortlink
3. Rename file according to shortlink and jpg extension (png file renamed to jpg also works)
4. Move file to avnotes folder
Especially the geolocation conversion is time consuming and error prone. Moving files to osmand's folder can also be complicated because new android versions hide the system folders in most contexts. Perhaps there could be an easier way: there could be an import button in the POI context menu or we could share an image with Osmand with an Intent and drag it to its location.

2. Suggestion: don't move map when turning tracks on or off
When turning a track on or off osmand will move and zoom the map to view the whole track region. I think it's not desirable. It would be reasonable to show this overview when a track is first imported but after that it's not necessary and for me a bit annoying, because the current position on the map is lost and takes time to find again. It's especially unnecessary to move the map when a subsection of the toggled track was already located on screen to begin with.

3. Bug: Sharing addresses creates multiple stacked search dialogs
Select an address in another app (e.g. in your browser) and share it with osmand. Now tap on one of the POIs found in search. Osmand will close the search dialog and open that POI on the map. Now go back to your browser and share another address, then tap on one of the found POIs again.
The expected result: the search screen should close and POI is shown on map, as before.
What happens: last search screen with the first address is shown again. Note the more often you repeat this process, ie. the more addresses you search with Share, the more stacked dialogs you get, so you need more and more taps on 'back' to return to the map.

4. Suggestion: render amenity=kneipp_water_cure
This tag is very widespread in germany. Unfortunately osmand currently doesn't render it and it's not available in search. So these places can't be found at all.
Note there is already a request on github (https://github.com/osmandapp/OsmAnd/issues/8372) about kneipp water cure, but it just says the icon and details are missing. The icon and details are not so important but we should be able to find these places.
Suggested names:
EN: Kneipp Water Cure
DE: Kneippbecken
The icon could be a basin icon with a health icon (cross) on top.

5. Bug: Touristic shelters are not rendered or searchable
In many cases an amenity=shelter is also tagged with a tourism= tag (e.g. viewpoint, picnic site, artwork...).
Example: https://www.openstreetmap.org/way/936281422
Osmand renders such nodes only as the Tourism node (e.g. viewpoint) with a Shelter: Yes property in the details. Unfortunately the shelter will not render on the map or show up in search. It's also not possible to create a custom filter e.g. for viewpoints with Shelter: Yes.
It would be nice to render such nodes as a Shelter instead, or render both Tourism and Shelter as separate POIs. In a worst case scenario, protection from the elements is much more important than having a nice view. In the general case, maybe all amenities should trump tourism when it comes to rendering priority.
Note on [osm.org](http://osm.org/), the above example is rendered as a shelter as expected.

6. Suggestion: Option to turn off animations
Android has an option called Remove Animations. It would be nice to respect this option in Osmand's dialogs or introduce another option inside osmand. Turning off animations will conserve some battery because there is less waiting and the screen is not redrawn unnecessarily.

7. Show bottle=yes information for amenity=drinking water
It would be nice to show this tag because it can be very useful not only for bottle filling. I have found it's also a good indicator for the reliability of the water source and the quality of the enclosure which can sometimes be an issue in remote areas.

**Comment by paulw517** [comment]
What is happening with Geolocation?
